blog/2023-03-10-conda-survey.md

```diff
@@ -0,0 +1,194 @@
+
+[![Which additional package managers do your use, top](/img/2023-03-10-conda-survey/which-additional-package-managers-do-you-use-top.png)](https://docs.google.com/spreadsheets/d/1rpWN6atSEsALvStPtAVIf_WMTxXXKaDKYpKRsvMpbOo/edit#gid=1102296811)
+
+79% of conda survey respondents use other package managers in addition to conda.  75% of all respondents also use pip, and 55% use mamba.
+
+The PSF survey asked "What Packaging tools do you use?"  They blended together package and environment management (as does conda itself):
+
+[![PSF: Which Python packaging tools do you use?](/img/2023-03-10-conda-survey/psf-which-python-packaging-tools-do-you-use.png)](/img/2023-03-10-conda-survey/psf-which-python-packaging-tools-do-you-use.png)
+(Note: PSF survey results are shown with a blue outline.)
+
+In this survey conda is used less frequently than 6 "standard" (and highly overlapping) entries (pip, venv/virtualenv, PyPI, setuptools, and wheel), and by poetry. pip has almost twice as many users as venv, the second most popular response. conda is followed closely by pipenv and twine.  conda-build is listed in the bottom half with 4-5% of respondents using it.
+
+(This is only one snapshot in time, so we can't say if the conda ecosystem is gaining or losing mind-share in the Python ecosystem.)
+
+The PSF survey asked several questions specifically related to interoperability.
+
+[![What should the packaging ecosystem do to be an "ecosystem for all"](/img/2023-03-10-conda-survey/psf-what-should-the-packaging-system-do-to-be;-an-ecosystem-for-all.png)](/img/2023-03-10-conda-survey/psf-what-should-the-packaging-system-do-to-be;-an-ecosystem-for-all.png)
+
+When PSF asked their respondents to rank these 4 priorities the highest ranked (by far) is "Support more interoperability between Python packaging tools."  At the very bottom was "Support interoperability between Python packaging and packaging tools for other languages."
+
+Several other questions revealed a strong preference for just one tool/workflow for package management.
+
+[![How much do you agree with these statements?](/img/2023-03-10-conda-survey/psf-how-much-do-you-agree-with-these-statements.png)](/img/2023-03-10-conda-survey/psf-how-much-do-you-agree-with-these-statements.png)
+
+The first question is "I prefer to have a clearly defined "official" workflow" which 75% agreed with.  A second similar (but inversely worded) question asks if "The existence of multiple tools is beneficial for the Python packaging ecosystem.  Here 46% disagree with the statement, and 23% are neutral
+
+**What do we do with all this?**
+
+Let's start by noting that the Python community strongly prioritizes interoperability between Python packaging tool, but that support for multiple languages is at the bottom.  This is good news and bad news for conda.  Conda already works well with Pip.  The downside is that most PSF respondents don't prioritize support for multiple languages, which is one of conda's superpowers.
```

Comment:
Consistency in `Pip` capitalization. Previous occurrences used `pip`. 

[`pypa/pip`](https://github.com/pypa/pip) uses `pip` (lowercase), always, even at the beginning of sentences.

`venv` should also go in lowercase, and maybe even backticked. [See docs](https://docs.python.org/3/library/venv.html).

Reply:
(Same applies in the paragraph below)

Reply:
Went for all lower case.  I did not backtick anything as then I would have to decide when to also backtick conda.